dylan-craig/styles/modules.css

```diff
@@ -0,0 +1,11 @@
+nav ul {
+  float: right;
+}
+
+nav li {
+  float: left;
+  padding-right: 2px;
+}
```

Comment:
The ul, and li in the nav are okay living in modules but I think this input should be moved to base, it's a general styling for the inputs on the whole page while ul and li point to more specific items within the nav.